yaml/builders/image-test.yaml

```diff
@@ -8,6 +8,14 @@
             #!/bin/bash
             set -ex
 
+            # Generate sources, commit changes into generated branch and show diff
+            ssh -F ssh_config host 'set -ex; \
+              cd sources; GIT_HASH=$(git rev-parse HEAD); \
+              if [ -f common/update.sh ]; then \
+                result="$(./common/update.sh)"; \
```

Comment:
If I understand this correctly - at this point we have the PR changes in local `master`, incl. container common scripts, and we are trying to check if any changes have been made to the generated sources. Is this correct?

If so then this needs to be changed a bit - the update script is expecting to be run from the `generated` branch. So maybe something like this?
```
if $(git checkout generated) # check if the branch even exists
  git submodule update # generated branch can (and probably will) be on a different common version
  result="$(./update)" # Possibly ./update $GIT_HASH in case $GIT_HASH is not HEAD of master
  git checkout -f $GIT_HASH && git submodule update; # back to the original branch and common scripts version
  [ "$result" = "Nothing changed" ] || git diff generated~1 generated -- $(sed -n 's/^VERSIONS[[:space:]]*=//p' Makefile) ;'
fi
```